### PR TITLE
PM Review 4 & adding UX

### PR DESCRIPTION
**Pull request by mcherem**
4th PM review, adding UX this time. Doc/product run thru.

### CODE REVIEW

File: modules/ROOT/pages/B2B-overview.adoc [hunk 1]
```diff
@@ -3,15 +3,18 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Partner Manager enables you to manage a large number of trading partners and their message flows. A message flow is the path along which a message travels, from its source, through a transformation (via a DataWeave map), and to its target.
+Anypoint Partner Manager enables you to manage a large number of trading partners and their message flows.
```
Comment (muledelp) [inline modules/ROOT/pages/B2B-overview.adoc]:
```suggestion
Anypoint Partner Manager enables you to manage the transmission of business messages with a large number of trading partners through the creation and deployment of message flows.
```

File: modules/ROOT/pages/B2B-overview.adoc [hunk 2]
```diff
@@ -3,15 +3,18 @@ ifndef::env-site,env-github[]
...
+A message flow is the path along which a message travels from its source, through a DataWeave transformation map, to its target.
```
Comment (muledelp) [inline modules/ROOT/pages/B2B-overview.adoc:7]:
```suggestion
A transmission is a single communication event during which a payload is transferred between you and one of your partners. These payloads can contain one or more messages. The message flows you create determine where the transmissions are received or sent and the path along which a message travels from its source, through a DataWeave transformation map, to its target. 
```

File: modules/ROOT/pages/B2B-overview.adoc [hunk 3]
```diff
@@ -3,15 +3,18 @@ ifndef::env-site,env-github[]
...
 
-You'll need to perform some tasks before you can begin transmitting messages with your trading partners. A transmission is a single communication event during which a payload is transferred between two business entities. These payloads can contain one or more business messages.
+You'll need to perform some tasks before you can begin transmitting messages with your trading partners.
+A transmission is a single communication event during which a payload is transferred between two business entities. These payloads can contain one or more business messages.
```
Comment (muledelp) [inline modules/ROOT/pages/B2B-overview.adoc:10]:
```suggestion
```

File: modules/ROOT/pages/_partials/endpoint-as2-send.adoc [hunk 4]
```diff
@@ -14,7 +14,7 @@ AS2 Send Settings pertain to trading partner-specific AS2 settings such as `Cert
 
 |The HTTP(S) address at which your partner is accepting incoming AS2 messages.
 The hostname here is dependent upon how you set up your host in Mule Runtime.
-The port is automatically generated when you deploy a message flow that uses this endpoint.  
+The port is automatically generated when you deploy a message flow that uses this endpoint.
 This derived URL is what you give to your partners to send to with AS2. +
```
Comment (muledelp) [inline modules/ROOT/pages/_partials/endpoint-as2-send.adoc]: Maria, it's confusing, we don't support AS2 Send yet. I think you want this to be an AS2 receive endpoint. If that's the case, then it should say "This derived URL is what you tell your partner is where you receive AS2 transmissions from them."

File: modules/ROOT/pages/_partials/endpoint-https-send.adoc [hunk 5]
```diff
@@ -1,5 +1,6 @@
 == HTTP(S) Protocol Send Settings
-Use these settings when sending with secure HTTP.
+
+Use these settings when sending with the secure HTTP protocol.
```
Comment (muledelp) [inline modules/ROOT/pages/_partials/endpoint-https-send.adoc]:
```suggestion
Use these settings when sending with the secure HTTP protocol. This includes endpoints for sending transmissions to your partner and endpoints that are targets at your organization.
```

File: modules/ROOT/pages/_partials/endpoint-http-send.adoc [hunk 6]
```diff
@@ -1,5 +1,6 @@
 == HTTP Protocol Send Settings
-Use these settings when sending with HTTP.
+
+Use these settings when sending with the HTTP protocol.
```
Comment (muledelp) [inline modules/ROOT/pages/_partials/endpoint-http-send.adoc:3]:
```suggestion
Use these settings when sending with the HTTP protocol. This includes endpoints for sending transmissions to your partner and endpoints that are targets at your organization.
```

File: modules/ROOT/pages/_partials/endpoint-sftp-send.adoc [hunk 7]
```diff
@@ -1,6 +1,6 @@
 == SFTP Protocol Send Settings
 
-Use these when setting up SFTP.
+Use these when setting up the SFTP protocol.
```
Comment (muledelp) [inline modules/ROOT/pages/_partials/endpoint-sftp-send.adoc]:
```suggestion
Use these when setting up the SFTP protocol. This includes endpoints for sending transmissions to your partner and endpoints that are targets at your organization.
```

File: modules/ROOT/pages/configure-message-flows.adoc [hunk 8]
```diff
@@ -3,105 +3,108 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To receive messages from a trading partner, create a message flow and configure the sections for both the source, mapping, and the target. Your organization is typically referred to as the host, while the enterprise you interact with is refered to as a trading partner.
+To receive messages from a trading partner, create a message flow and configure the sections for the source, target, and mapping.
+Your organization is typically referred to as the host. The enterprise you interact with is referred to as your trading partner.
 
 For the receiving endpoint - this is the origin of the message flow when defined by a host endpoint and message type, often an API.
```
Comment (muledelp) [inline modules/ROOT/pages/configure-message-flows.adoc]:
```suggestion
The source of the message flow is defined by the partner, a host endpoint, often an API, and the message type you are receiving from them.
```

File: modules/ROOT/pages/configure-message-flows.adoc [hunk 9]
```diff
@@ -3,105 +3,108 @@ ifndef::env-site,env-github[]
...
 
 For the target endpoint - this is the final destination of a message flow when defined by a host endpoint and message type, often a Host API.
```
Comment (muledelp) [inline modules/ROOT/pages/configure-message-flows.adoc]:
```suggestion
The target is the final destination of a message flow and is defined by a host message type and endpoint, often a Host API.
```

File: modules/ROOT/pages/configure-message-flows.adoc [hunk 10]
```diff
@@ -3,105 +3,108 @@ ifndef::env-site,env-github[]
...
+* The trading partner's name and associated identifiers (such as X12 ISA, X12 GS, and so on).
 
-* The endpoint where you receive the transmission
+* The endpoint where you receive the transmission.
 
-* The message type to process for this message flow (such as X12, 4010, 850, and so on)
+* The message type to process for this message flow (such as X12, 4010, 850, and so on).
 
-* The endpoint where your trading partner may want to receive an acknowlegement related to the transactions they send
+* The endpoint where your trading partner may want to receive an acknowledgment related to the transactions they send.
```
Comment (muledelp) [inline modules/ROOT/pages/configure-message-flows.adoc]:
```suggestion
* The endpoint where your trading partner may want to receive an acknowledgment related to the messages they send.
```

**Review by muledelp — CHANGES_REQUESTED** [review]
I couldn't review all the files and don't know all the technical stuff anyway. Configure message flows could use another pass. Plus, I slacked you some of the comments or suggestions I couldn't make in github.
From Maria: I have addressed all your comments. Please approve when possible.

File: modules/ROOT/pages/_partials/endpoint-as2-send.adoc (hunk 4 above)
Comment (mcherem) [reply modules/ROOT/pages/_partials/endpoint-as2-send.adoc]: Nope, I don't want to show this one.  I will hide this for now and add the AS2 Receive doc.

File: modules/ROOT/pages/B2B-overview.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/B2B-overview.adoc]: Fixed.

File: modules/ROOT/pages/_partials/endpoint-as2-send.adoc (hunk 4 above)
Comment (mcherem) [reply modules/ROOT/pages/_partials/endpoint-as2-send.adoc]: New AS2 and SFTP receive files have been added.

File: modules/ROOT/pages/_partials/endpoint-http-send.adoc (hunk 6 above)
Comment (mcherem) [reply modules/ROOT/pages/_partials/endpoint-http-send.adoc:3]: Fixed.

File: modules/ROOT/pages/B2B-overview.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/B2B-overview.adoc:7]: You suggested def has a very long sentence at  the end. I can try to condense it, but we could end up in a long debate about it. Plus, it's different from what Amit has sanctioned. I've ask you both to come to an agreement on the definition.

File: modules/ROOT/pages/B2B-overview.adoc (hunk 3 above)
Comment (mcherem) [reply modules/ROOT/pages/B2B-overview.adoc:10]: same as above.

File: modules/ROOT/pages/B2B-overview.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/B2B-overview.adoc:7]: Will use Amit's def per David.

File: modules/ROOT/pages/B2B-overview.adoc (hunk 3 above)
Comment (mcherem) [reply modules/ROOT/pages/B2B-overview.adoc:10]: Will use Amit's def per David's okay.

**Review by amitfernandes — APPROVED** [review]
Approving PR

**Review by muledelp — APPROVED** [review]

**Pull request merged**